packages/core/src/components/Button/types.ts

```diff
@@ -12,6 +12,7 @@ export interface IButtonProps {
   className?: string;
   onClick?: () => void;
   children?: React.ReactNode;
+  [key: `aria-${string}`]: string | number | boolean;
```

Comment:
I suggest to use specific aria-* keys, because in this case we can write anything for the aria-* property name. In another side, we can extends from all button props.
Please look to the `AriaAttributes `interface. I think, we can extend from it (if we need only additional aria attributes)

Reply:
Thanks for pointing out the `AriaAttributes` interface. I will use it here.